### PR TITLE
Switch frugal binary and library to use glide instead of godep

### PR DESCRIPTION
**Pull request by charliestrawn-wf**
This seems to be working, need to fix the travis python permissions issues

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

File: scripts/skynet/run_cross_skynet.sh [hunk 1]
```diff
@@ -25,6 +25,8 @@ frugal --gen py:asyncio -r --out='test/integration/python/asyncio/gen_py_asyncio
 frugal --gen py -r -out='test/integration/python/tornado/gen-py' test/integration/frugalTest.frugal
 frugal --gen dart:use_enums=true -r --out='test/integration/dart/gen-dart' test/integration/frugalTest.frugal
 
+go get github.com/Sirupsen/logrus
```
Comment (brianshannan-wf) [inline scripts/skynet/run_cross_skynet.sh]: Why do we need to explicitly go get this?

Comment (charliestrawn-wf) [reply scripts/skynet/run_cross_skynet.sh]: The tests use it, but they are run from a test/integration dir. Frugal doesn't use it, so when it walks the gopath it never finds it. I can look at just using the go library logger for integration tests going forward.

**Comment by brianshannan-wf** [comment]
@aviary-wf

File: lib/go/glide.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package: github.com/Workiva/frugal/lib/go
+import:
+- package: git.apache.org/thrift.git
+  version: 0d9b713b173f35ce02552b2f4372899440a99b25
```
Comment (brianshannan-wf) [inline lib/go/glide.yaml]: Should we pin this to an actual tag release? Same question for others below

File: scripts/skynet/run_cross_skynet.sh (hunk 1 above)
Comment (brianshannan-wf) [reply scripts/skynet/run_cross_skynet.sh]: Doesn't that directory have it's own glide.yaml?

**Comment by rmconsole-wf** [comment]
@charliestrawn-wf This pull request has merge conflicts, please resolve.

**Comment by brianshannan-wf** [comment]
Closing in favour of https://github.com/Workiva/frugal/pull/976

**Pull request closed without merging**